install_config/install/advanced_install.adoc

```diff
@@ -589,6 +595,84 @@ openshift_master_overwrite_named_certificates=true
 ----
 ====
 
+[[advanced-install-cluster-metrics]]
+== Configuring Cluster Metrics
+
+Cluster metrics are not set to automatically deploy by default. Set the
+following to enable cluster metrics when using the advanced install:
+
+====
+----
+[OSEv3:vars]
+
+openshift_hosted_metrics_deploy=true
+----
+====
+
```

Comment:
How does this relate to/affect the existing documentation on cluster metrics? Should they be linked? Is the old stuff still relevant?